cgi-bin/image_resizer_web.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+import cgi, os
+from PIL import Image
+
+print("Content-type: text/html")
+print("<meta charset=\"utf-8\">")
+print()
+print("<style>h1,p{text-align:center;}</style>")
+
+# Get the data
+field = cgi.FieldStorage()
+path_in = field.getfirst("path_in")
+width = field.getfirst("width")
+path_out = field.getfirst("path_out")
+path_water = field.getfirst("path_water")
+pos_water = field.getfirst("pos_water")
+
+# Check if options are exist
+if path_in is None:
+    path_in = "/Users/garazd/Documents/Python/Img_In/"
+if path_out is None:
+    path_out = "/Users/garazd/Documents/Python/Img_Out/"
+if width is None:
+    width = 600
```

Comment:
I had a problem with the default parameter of getfirst() method. If the field is empty it returns an empty string instead of the default value. Does anyone have this problem too?